packages/nodes-base/nodes/CrowdDev/CrowdDevNode.node.json

```diff
@@ -0,0 +1,18 @@
+{
+	"node": "n8n-nodes-base.crowd",
```

Comment:
Like @janober said in https://github.com/n8n-io/n8n/pull/6082#issuecomment-1537916247 Is this supposed to be `crowdDev` right?